tests/plugins/project/test_extension_helpers.py

```diff
@@ -704,7 +704,7 @@ def test_bump_runner_version(mocker, capsys):
         assert 'Runner version has been successfully updated to 1.0' in captured.out
         assert f'{os.path.join(project_dir, "docker-compose.yml")}' in captured.out
         assert f'{os.path.join(project_dir, "OtherDockerfile")}' in captured.out
-        assert f'{os.path.join(project_dir, "Dockerfile")}' in captured.out
```

Comment:
The captured line in long and Docker file name is separated by new line, so left just so